packages/destination-actions/src/destinations/marketo-static-lists/addToList/index.ts

```diff
@@ -16,13 +16,76 @@ const action: ActionDefinition<Settings, Payload> = {
     batch_size: { ...batch_size },
     event_name: { ...event_name }
   },
-  perform: async (request, { settings, payload, statsContext }) => {
+  hooks: {
+    retlOnMappingSave: {
+      label: 'Create a new static list in Marketo',
```

Comment:
Maybe this since they can create or connect to an existing list?
```suggestion
      label: 'Connect to a static list in Marketo',
```